<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonalMain</class>
 <widget class="QDialog" name="PersonalMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-color: rgb(211,211,211);
	border-top-right-radius : 15px;
	border-bottom-right-radius: 15px;
}
QButton
{
	background-color: rgb(255,255,255);
}
QLabel
{
	font: 9pt &quot;微软雅黑&quot;;
}

</string>
  </property>
  <widget class="QComboBox" name="statusEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>在线</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>隐身</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>离线</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="headImg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 188, 19);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="signatureEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9.07563pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9.07563pt;&quot;&gt;（签名）&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>（名字）</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>450</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>7</number>
    </property>
    <item>
     <widget class="QPushButton" name="btnConfirm">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	font: 9pt &quot;微软雅黑&quot;;
	border-radius :7px;
	background-color: rgb(255,255,255);
}
QPushButton:hover
{
	background-color: rgb(169,189,35);
}</string>
      </property>
      <property name="text">
       <string>修改信息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPassword">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	font: 9pt &quot;微软雅黑&quot;;
	border-radius :7px;
	background-color: rgb(255,255,255);
}
QPushButton:hover
{
	background-color: rgb(169,189,35);
}</string>
      </property>
      <property name="text">
       <string>修改密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnQuit">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	font: 9pt &quot;微软雅黑&quot;;
	border-radius :7px;
	background-color: rgb(255,255,255);
}
QPushButton:hover
{
	background-color: rgb(169,189,35);
}</string>
      </property>
      <property name="text">
       <string>退出登录</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>301</width>
     <height>349</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="accountLabel">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>账   号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="accountEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="ageLabel">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>年   龄</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ageEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="genderLabel">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>性   别</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="genderEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="currentText">
           <string>-请选择-</string>
          </property>
          <item>
           <property name="text">
            <string>-请选择-</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>男</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>女</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="cityLabel">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>城   市</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cityEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>-请选择-</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>北京</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>天津</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>河北</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>辽宁</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>吉林</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>黑龙江</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>山东</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>江苏</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>上海</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>浙江</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>安徽</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>福建</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>江西</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>广东</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>广西</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>海南</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>河南</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>湖南</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>湖北</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>山西</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>内蒙古</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>宁夏</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>青海</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>陕西</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>甘肃</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新疆</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>四川</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>贵州</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>云南</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>重庆</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>西藏</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>香港</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>澳门</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>台湾</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="jobLabel">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>职   位</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="jobEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>-请选择-</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>教师</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>学生</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>公务员</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>律师/法务</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>计算机/通信</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>广告/传媒</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>医疗/护理</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>其他</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="passwordOldLabel">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>旧密码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordOldEdit">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLineEdit" name="passwordNewEdit">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="passwordNewLabel">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>新密码</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="passwordConfirmLabel">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>确认密码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordConfirmEdit">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>331</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 9pt &quot;微软雅黑&quot;;
	border-radius :7px;
	background-color: rgb(255,255,255);
}
QPushButton:hover
{
	background-color: rgb(169,189,35);
}</string>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <zorder>statusEdit</zorder>
  <zorder>headImg</zorder>
  <zorder>signatureEdit</zorder>
  <zorder>nameEdit</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>line</zorder>
  <zorder>btnSave</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
